charts/orc/Chart.yaml

```
@@ -0,0 +1,21 @@
+# Copyright Copyright 2025 The ORC Authors.
+# SPDX-License-Identifier: APACHE-2.0
+
+apiVersion: v2
+name: orc
+type: application
+version: 0.1.1
+appVersion: 2.0.3
+kubeVersion: ">=1.29.0-0"
+description: A Helm chart for https://github.com/k-orc/openstack-resource-controller/
+home: https://k-orc.cloud/
+keywords:
+  - openstack
+  - infrastructure
+sources:
+  - https://https://github.com/k-orc/openstack-resource-controller/tree/main/charts/orc
```

URL is off there too
```suggestion
  - https://github.com/k-orc/openstack-resource-controller/tree/main/charts/orc
```